internal/server/server.go

```diff
@@ -79,6 +80,11 @@ func createSOMASServer(
 			CommonPool:  gameConfig.InitialCommonPool,
 		},
 	}
+	if gameConfig.IIGOConfig.StartWithRulesInPlay {
+		for ruleName, _ := range rules.AvailableRules {
```

Comment:
can you see linter warnings? seems like most people can't because they are not fixed

Reply:
XD, as in it doesn't show in IDE, I have to look at PR's to see them. Fixed in latest push